src/client/app/strings/english.ts

```diff
@@ -75,6 +75,8 @@ export function getEnglishStrings(): ClientStrings {
         gistSharedDialogEnd: `To import your snippet, select the Import button in ${playgroundName} and enter this URL.`,
         gistSharedDialogTitle: 'Share your snippet',
         gistSharedDialogViewButton: 'View on GitHub',
+        gistUpdatedDialog: /** NEEDS STRING REVIEW */ 'The URL of your updated gist is the same as before:',
```

Comment:
I think the name of the string should more closely correspond to the title of the text.  I.e., `gistUpdateUrlIsSameAsBefore`